purchase_archive/README.rst

```diff
@@ -0,0 +1,16 @@
+Purchase Archive
```

Comment:
Please use the OCA template for the readme: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Reply:
As far as I understand I just have to add the files into the readme folder and the README.rst is automatically generated?

Reply:
Or do I have to execute somehow gen_addon_readme.py?